### ISSUE TITLE
Extension records incorrect net_amount

### ISSUE THREAD

**Issue by MegaphoneJon** [JMAConsulting/biz.jmaconsulting.lineitemedit#59]
Without this extension installed, if I record a contribution (manually) for $10, with a $2 `fee_amount`, the `net_amount` will be $8.

[This line in the extension](https://github.com/JMAConsulting/biz.jmaconsulting.lineitemedit/blob/372de878adf8c3a7886ac21582a600612c7b9ba4/lineitemedit.php#L218) will set my `net_amount` to $10, so my `fee_amount` + `net_amount` is $12.

I'm not sure why `net_amount` is being referenced here.  Is there any harm in removing the reference to it?

**Comment by MegaphoneJon** [comment]
As a follow-up; to fix errors introduced here, is there any harm in running this?
```
UPDATE civicrm_contribution SET net_amount = total_amount - fee_amount;
```

It seems (to my layperson eyes) that the financial transactions are recorded correctly despite this issue, so I think it's more appropriate to fix this via direct SQL.

**Comment by JoeMurray** [comment]
Hi Jon, thanks for this report, and sorry for the error.

Monish, can you figure out if this was ever correct in the extension?

**Comment by seamuslee001** [comment]
PR has been merged and fix released in 2.4 closing
